Ghidra/Framework/Graph/src/main/java/ghidra/graph/graphs/FilteringVisualGraph.java

```diff
@@ -118,7 +118,7 @@ public Iterator<V> getFilteredVertices() {
 		Predicate<? super V> isFiltered = v -> {
 			return !containsVertex(v) && completeGraph.containsVertex(v);
 		};
-		return Iterators.filter(getAllVertices(), isFiltered);
+		return Iterators.filter(getAllVertices(), isFiltered::test);
```

Comment:
There shouldn't be a need to use Guava's `Iterators.filter()` method anymore.  It's a little confusing too.  You should be able to use Java Streams.